### ISSUE TITLE
KeyError: 'page' when creating new page

### ISSUE THREAD

**Issue by stuartbman** [wagtail/wagtail-review#41]
I can edit existing pages, but on creating new pages, I get the following error:
```
KeyError
'page'
```
from:
```
# editor where we also have the page ID
class GuacamoleMenuItem(ActionMenuItem):
    def render_html(self, request, context):
        return mark_safe(f"<script>window.wagtailPageId = {context['page'].id};</script>")
```

Wagtail version 2.10

**Comment by gasman** [comment]
Thanks @stuartbman! Looks like this is the issue that was previously reported and fixed in #22, but it ended up getting rolled back when the integration with Wagtail 2.10's workflow features didn't happen... re-fixing now, and will get a patch release out shortly.

**Comment by stuartbman** [comment]
Thanks very much, that's great!
